centreon/www/class/centreonDowntime.class.php

```diff
@@ -373,18 +373,18 @@ public function getForEnabledHosts()
                 h.host_name,
                 NULL as service_id,
                 NULL as service_description
-            FROM downtime_period dtp,
-                downtime dt,
-                downtime_host_relation dtr,
-                host h
-            WHERE dtp.dt_id = dtr.dt_id
-                AND dtp.dt_id = dt.dt_id
-                AND dtr.host_host_id = h.host_id
-                AND h.host_activate = '1'
+            FROM downtime_period dtp
+            INNER JOIN downtime dt 
+                ON dtp.dt_id = dt.dt_id
+            INNER JOIN downtime_host_relation dtr 
+                ON dtp.dt_id = dtr.dt_id
+            INNER JOIN host h 
+                ON dtr.host_host_id = h.host_id
+            WHERE h.host_activate = '1'
         SQL;
 
-        $statement = $this->db->query($request);
-
+        $statement = $this->db->prepare($request);
+        $statement->execute();
```

Comment:
There's nothing to prepare here.
```suggestion
        $statement = $this->db->query($request);
```